UpdateReferences.bat

```diff
@@ -1,4 +1,4 @@
 @Echo Off
 cd TMG-Framework
-git pull --recurse-submodules=on origin dev
```

Comment:
Just a minor note - might be useful to have bash / shell script for this as well.